endoreg_db/data/ai_model_label/label/polyp_classification.yaml

```
@@ -0,0 +1,51 @@
+    name: "nice_2a"
+    label_type: "classification"
+
+- model: endoreg_db.Label
+  fields:
+    name: "nice_2b"
+    label_type: "classification"
+
+- model: endoreg_db.Label
+  fields:
+    name: "nice_3"
+    label_type: "classification"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)